tabpfn/scripts/transformer_prediction_interface.py

```diff
@@ -421,7 +455,8 @@ def get_preprocess(i):
                 #                        preprocess_input(eval_xs_, preprocess_transform='none')], -1)
                 #eval_xs_ = torch.flatten(torch.swapaxes(eval_xs_, -2, -1), -2)
             else:
-                eval_xs_ = preprocess_input(eval_xs_, preprocess_transform=preprocess_transform_configuration)
+                eval_xs_ = preprocess_input(eval_xs_, preprocess_transform=preprocess_transform_configuration,
```

Comment:
You can remove the `combine_preprocessing` and then I believe this to be the only call to pre_process_input, right? Then we can take the detach out of preprocess_input, have it here, and remove the `no_grad` argument. Could you do that?

Reply:
Yeah, sure. I will delete the upper if part and remove all occurrences of combine_preprocessing. This is then the only call to preprocess_input, and I will change it as you've described it.